src/fluree/db/query/exec/select/display.cljc

```diff
@@ -0,0 +1,65 @@
+(ns fluree.db.query.exec.select.display
+  (:require [fluree.db.constants :as const]
+            [fluree.db.query.exec.where :as where]
+            [fluree.db.util.json :as json]))
+
+(defmulti fql (fn [match compact] (where/get-datatype-iri match)))
```

Comment:
I think this should also be split. Instead of this new namespace with these two multimethods, we should instead put the `display/fql` multimethod in `fluree.db.query.exec.select.fql` and the sparql multimethod in the sparql ns.

Essentially, we should take everything except for the protocol definitions from `fluree.db.exec.select.fql` and move them to a new namespace `fluree.db.query.exec.select.fql`, and then define a parallel `fluree.db.query.exec.select.sparql` namespace with the sparql functionality.